setup.py

```diff
@@ -103,10 +129,53 @@ def get_version():
             'Programming Language :: Python :: 3.6',
         ],
         license='Apache License 2.0',
-        setup_requires=['pytest-runner'],
+        setup_requires=['pytest-runner', 'cython', 'torch>=1.1', 'numpy'],
         tests_require=['pytest'],
         install_requires=[
             'mmcv>=0.2.6', 'numpy', 'matplotlib', 'six', 'terminaltables',
-            'pycocotools'
+            'pycocotools', 'torch>=1.1'
+        ],
+        ext_modules=[
+            make_cython_ext(
+                name='soft_nms_cpu',
+                modpath=('mmdet', 'ops', 'nms'),
```

Comment:
Why not just use `mmdet.ops.nms` so that we do not need `'.'.join(modpath)`.

Reply:
Because it's also needed to construct the path to the source files. Alternatively, we can pass in `mmdet.ops.nms` and split on '.'